.github/workflows/2_auto_publish_release.yml

```diff
@@ -37,6 +37,7 @@ jobs:
       run: |
         sudo apt-get install -y graphviz
         pip install git+https://github.com/cylc/cylc-flow/  # install latest cylc-flow
+        pip instal wheel
```

Comment:
Instead of this (would need to be added to the stage 1 workflow as well), what about adding it as the first step of the "build-python-package" action itself? https://github.com/cylc/release-actions/blob/master/build-python-package/action.yml

Reply:
Seems sensible.

Reply:
https://github.com/cylc/release-actions/pull/7